website/public/js/services/groupServices.js

```diff
@@ -94,10 +94,10 @@ angular.module('habitrpg')
       });
     };
 
-    Group.startQuest = function(gid) {
+    Group.startQuest = function(gid, key) {
       return $http({
         method: "POST",
-        url: groupApiURLPrefix + '/' + gid + '/questAccept',
+        url: groupApiURLPrefix + '/' + gid + '/quests/invite/' + key,
```

Comment:
Yea, the invite route used to be in the accept route. So, I didn't realize that for a while and did not update the function names.